src/MoBi.Presentation/Presenter/HierarchicalSimulationPresenter.cs

```diff
@@ -1,40 +1,45 @@
-using System;
-using System.Collections.Generic;
-using System.Drawing;
-using System.Linq;
 using MoBi.Assets;
-using OSPSuite.Utility.Events;
-using OSPSuite.Utility.Extensions;
 using MoBi.Core.Domain.Model;
 using MoBi.Core.Events;
 using MoBi.Presentation.DTO;
 using MoBi.Presentation.Mappers;
 using MoBi.Presentation.Nodes;
 using MoBi.Presentation.Views;
 using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Services;
 using OSPSuite.Core.Extensions;
 using OSPSuite.Presentation.Core;
 using OSPSuite.Presentation.Presenters;
 using OSPSuite.Presentation.Presenters.ContextMenus;
+using OSPSuite.Utility.Events;
+using OSPSuite.Utility.Extensions;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Windows.Forms;
```

Comment:
Can you check that these are all necessary? I'm especially worried about dependency on System.Windows.Forms, but sometimes they get added during refactoring and then become stale. Check all the files you changed please.

We normally don't have a dependency on System.Windows.Forms in the Presentation layer.

Reply:
It's because of the Clipboard static class I use now in the CopyCurrentPathToClipBoard-method.

Reply:
I also added OSPSuite.Core.Domain.Services for the EntityPathResolver. No other usings added.

Reply:
We should move the ClipBoard code into the UI layer as discussed so that the dependency on Forms can be moved.

Reply:
Moved the clipboard usage to the view and thus removed the dependency on System.Windows.Forms from the Presentation layer.